### PR TITLE
DE19399:Volume capacity attribute size/unit problems

### PR DESCRIPTION
**Pull request by ron-saito**
Fixes converting the volume capacity from GB to GiB for volume create and update, and from GiB to KiB for volume get

### CODE REVIEW

File: internal/resources/resource_volume.go [hunk 1]
```diff
@@ -315,8 +319,13 @@ func resourceMetalVolumeRead(d *schema.ResourceData, meta interface{}) (err erro
 		return err
 	}
 
+	// convert from KiB to GB
```
Comment (dbozzato81) [inline internal/resources/resource_volume.go:322]: Move comment after line 323.

**Review by dbozzato81 — COMMENTED** [review]
LGTM. One minor comment.

File: internal/resources/resource_volume.go [hunk 2]
```diff
@@ -36,6 +37,9 @@ const (
 	vID          = "id"
 	vDiscoveryIP = "discovery_ip"
 	vTargetIQN   = "target_iqn"
+
+	GBToGiBConversion float64 = 0.931323
```
Comment (mchuang3) [inline internal/resources/resource_volume.go:41]: Not sure why this is rounded to 0.931323. Math says it's 0.931322574615478515625, so should be 0.931322. Google shows a few places doing 0.931323. Not sure I agree, but probabaly doesn't matter.

Comment (ron-saito) [reply internal/resources/resource_volume.go:41]: 0.931322574615478515625 rounded to 6 places is 0.931323

**Review by mchuang3 — APPROVED** [review]

**Pull request merged**